wolfssl/ssl.h

```diff
@@ -4063,7 +4075,11 @@ WOLFSSL_API int wolfSSL_SSL_CTX_remove_session(WOLFSSL_CTX *,
 WOLFSSL_API WOLFSSL_BIO *wolfSSL_SSL_get_rbio(const WOLFSSL *s);
 WOLFSSL_API WOLFSSL_BIO *wolfSSL_SSL_get_wbio(const WOLFSSL *s);
 WOLFSSL_API int wolfSSL_SSL_do_handshake(WOLFSSL *s);
+#if defined(WOLFSSL_QT)
```

Comment:
Switch on OPENSSL_VERSION_NUMBER instead of WOLFSSL_QT

Reply:
Guard by OPNESSL_VERSION_NUMBER instead of WOLFSSL_QT